Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -341,9 +341,8 @@ void CFrame::DoOpen(bool Boot)
   wxString path = wxFileSelector(
       _("Select the file to load"), wxEmptyString, wxEmptyString, wxEmptyString,
       _("All GC/Wii files (elf, dol, gcm, iso, tgc, wbfs, ciso, gcz, wad)") +
-          wxString::Format(
-              "|*.elf;*.dol;*.gcm;*.iso;*.tgc;*.wbfs;*.ciso;*.gcz;*.wad;*.dff;*.tmd|%s",
-              wxGetTranslation(wxALL_FILES)),
+          wxString::Format("|*.elf;*.dol;*.gcm;*.iso;*.tgc;*.wbfs;*.ciso;*.gcz;*.wad;*.dff|%s",
```

Comment:
Just FYI, opening a TMD from User\Wii\title\* (e.g. User\Wii\title\title\00000001\00000002\content\title.tmd for the 
Sys Menu) worked the same as booting the WAD of such title directly. Can't confirm if that's still how it worked before this commit though, especially after the recent boot code cleanups...